learning/index.html

```diff
@@ -1,9 +1,18 @@
 ---
-title: External resources
-description: Here is a collection of learning material from third parties that might help you master the language.
+title: Learning
+description: Here is a collection of learning material that can help you get familiar and master the language.
 ---
 <div class="container">
 
+  {% include community_row.html
+    title="Language Introduction"
+    divId="tutorial-introduction"
+    content="This course is targeted at novice Crystal programmers and conveys a basic understanding of the language's core concepts. Prior programming experience is recommended, but not required. This course can help you learn your first programming language."
+    link_text="Tutorial: Language Introduction"
+    url="https://crystal-lang.org/reference/1.6/tutorials/basics/index.html"
```

Comment:
There's no point in being specific about the version
```suggestion
    url="https://crystal-lang.org/reference/tutorials/basics/index.html"
```